src/mixins/view.js

```diff
@@ -142,6 +141,7 @@ const ViewMixin = {
     this._deleteEntityEventHandlers();
 
     this.triggerMethod('destroy', this, options);
+    this._triggerEventOnBehaviors('destroy', this, options);
```

Comment:
It will not duplicate destroy triggerMethod call in behaviors?

Reply:
the listeners on the behavior are cleaned up by this point, so the destroy needs to be handled manually.  Not true for `before:destroy` however